app/models/reservation.rb

```diff
@@ -103,6 +103,25 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  def self.deletable_missed
```

Comment:
I think it belongs more here than in the job -- the jobs shouldn't know the details of what they delete/otherwise handle, just that there's this method that will grab the right collection (right?). 

I'll go ahead and make them query objects